docker/images/kuma_base/Dockerfile

```diff
@@ -10,22 +10,22 @@ ENV PYTHONDONTWRITEBYTECODE=1 \
 RUN set -x \
     && apt-get update \
     && apt-get install -y --no-install-recommends \
-        curl \
-        gpg \
-        dirmngr \
-        libsasl2-modules \
-        gettext \
-        mime-support \
-        build-essential \
-        libtidy-dev \
-        libxml2-dev \
-        libxslt1-dev \
-        libffi-dev \
-        libjpeg-dev \
-        libmagic-dev \
-        default-libmysqlclient-dev \
-        default-mysql-client \
-        python2 \
+    curl \
+    gpg \
+    dirmngr \
+    libsasl2-modules \
+    gettext \
+    mime-support \
+    build-essential \
+    libtidy-dev \
+    libxml2-dev \
+    libxslt1-dev \
+    libffi-dev \
+    libjpeg-dev \
+    libmagic-dev \
+    default-libmysqlclient-dev \
+    default-mysql-client \
+    python2 \
```

Comment:
I'm sorry I didn't notice this. My VS Code (apparently) has a formatted for `Dockerfile`s. And it applied this formatting fix. I swear I didn't change anything. Just the VS Code formatting.